runtime/src/main/scala/grpcgateway/handlers/SwaggerHandler.scala

```diff
@@ -86,7 +86,8 @@ class SwaggerHandler(services: Seq[GrpcGatewayHandler]) extends ChannelInboundHa
   private val mimeTypes = new MimetypesFileTypeMap()
   mimeTypes.addMimeTypes("image/png png PNG")
   mimeTypes.addMimeTypes("text/css css CSS")
-  private val serviceUrls  = services.map(s => s"{url: '/specs/${s.name}.yml', name: '${s.name}'}").mkString(", ")
+  // Hardcode generated swagger yml file name to ServiceService.yml since this is the defined package name for GRPC services
+  private val serviceUrls  = services.map(s => s"{url: '/specs/ServiceService.yml', name: '${s.name}'}").mkString(", ")
```

Comment:
i think this must be solved the other way round. rather than hardcoding the name we should be able to pick the service name yml when serving the swagger docs.